### PR TITLE
Adding support for configuring operations

### PR DESCRIPTION
**Pull request by sudomateo**
Attempting to add the ability to manage execution mode via the SDK. Looking for some feedback and also help with adding this to tests.

### CODE REVIEW

**Comment by sudomateo** [comment]
Thanks for the feedback. I added this to existing tests, but I didn't write a new test as I would need some more guidance there.

**Comment by svanharmelen** [comment]
When/if all the checks pass (I can no longer test this easily myself), this should be good to go 👍

File: workspace_test.go [hunk 1]
```diff
@@ -376,6 +383,7 @@ func TestWorkspacesUpdateByID(t *testing.T) {
 			assert.Equal(t, *options.AutoApply, item.AutoApply)
 			assert.Equal(t, *options.FileTriggersEnabled, item.FileTriggersEnabled)
 			assert.Equal(t, *options.QueueAllRuns, item.QueueAllRuns)
+			assert.Equal(t, *options.Operations, item.Operations)
```
Comment (svanharmelen) [inline workspace_test.go]: In order to keep things (more or less) consistent, this should be moved above `QueueAllRuns` everywhere in this file 😏

Comment (sudomateo) [reply workspace_test.go]: Oops! Clearly I can't alphabet. Going to make the changes soon. I'm also in the process of working with a developer regarding the CI builds.

**Pull request merged**